### PR TITLE
feat(create): Add possibility to create custom series

### LINKED ISSUE #13: Ability to create a new series withe defined values
When I draw a graph with the result of a TSL query, I often need to add a custom series which can be for example an upper limit.

Is it possible to do it in TSL to perform the two operations at the same time

Exemple:
```
create(series('metrics.service.mads').setLabels('service=abcd').addValue(now, 150).addValue(now - 1h, 150)...) 
```

### PR DESCRIPTION
**Pull request by aurrelhebert**
fix #13

### CODE REVIEW

File: tsl/generateWarpScript.go [hunk 1]
```diff
@@ -414,6 +428,71 @@ func (protoParser *ProtoParser) operatorBy(framework FrameworkStatement, hasBy b
...
+		buffer.WriteString("\n")
+
+		//timePrefix := createSeries.end
+
+		for _, value := range createSeries.values {
+			buffer.WriteString(prefix + "    ")
+
+			tick := value.tick.lit
+
+			log.Warn(tick)
```
Comment (miton18) [inline tsl/generateWarpScript.go]: warn ?

File: tsl/generateWarpScript.go [hunk 2]
```diff
@@ -414,6 +428,71 @@ func (protoParser *ProtoParser) operatorBy(framework FrameworkStatement, hasBy b
...
+			if value.tick.tokenType == DURATIONVAL {
+				tick = protoParser.parseShift(value.tick.lit)
+			}
+
+			if createSeries.end.lit != "" {
+
+				end := createSeries.end.lit
+
+				if createSeries.end.tokenType == STRING && createSeries.end.lit == "now" {
```
Comment (miton18) [inline tsl/generateWarpScript.go]: Do you thing it can be easier if `now` is defined for the whole script ?

**Review by miton18 — CHANGES_REQUESTED** [review]

File: tsl/genericParser.go [hunk 3]
```diff
@@ -846,6 +867,231 @@ func (p *Parser) parseConnect(tok Token, pos Pos, lit string, instruction *Instr
...
+				errMessage := fmt.Sprintf("Unvalid param found in setValue expect a tick as Number or duration, found %q", tokstr(valueTok, valueLit))
+				return nil, nil, p.NewTslError(errMessage, valuePos)
+			}
+			valueField := &InternalField{tokenType: valueTok, lit: valueLit}
+
+			dataPoint := DataPoint{tick: tickField, value: valueField}
+
+			createSeries.values = append(createSeries.values, dataPoint)
+
+			log.Warn(dataPoint.tick)
```
Comment (miton18) [inline tsl/genericParser.go]: warn ?

File: tsl/genericParser.go [hunk 4]
```diff
@@ -846,6 +867,231 @@ func (p *Parser) parseConnect(tok Token, pos Pos, lit string, instruction *Instr
...
+			log.Warn(dataPoint.value)
```
Comment (miton18) [inline tsl/genericParser.go]: warn

**Review by miton18 — APPROVED** [review]

**Review by PierreZ — APPROVED** [review]

**Pull request merged**